compiler/src/dmd/expressionsem.d

```diff
@@ -17757,10 +17757,11 @@ void lowerNonArrayAggregate(StaticForeach sfe, Scope* sc)
     Parameters*[3] pparams = [new Parameters(), new Parameters(), new Parameters()];
     foreach (i; 0 .. nvars)
     {
-        foreach (params; pparams)
+        foreach (j, params; pparams)
         {
             auto p = sfe.aggrfe ? (*sfe.aggrfe.parameters)[i] : sfe.rangefe.param;
-            params.push(new Parameter(aloc, p.storageClass, p.type, p.ident, null, null));
+            auto storageClass = j == 2 ? p.storageClass & ~STC.manifest & ~STC.alias_ : p.storageClass;
```

Comment:
```suggestion
            auto storageClass = j == 2 ? p.storageClass & ~(STC.manifest | STC.alias_) : p.storageClass;
```
Chose whichever you think is more understandable.

Reply:
I don't have a strong preference.